config/locales/en.yml

```
@@ -15,3 +15,8 @@ en:
       edit: Edit
       destroy: Destroy
       delete_confirmation: Are you sure?
+  controllers:
+    notice:
+      create: successfully created.
```

```suggestion
      create: %{name} was successfully created.
```

みたいにすると動的にメッセージを変更できることができます。
これを使ってみてください〜。